### DISCUSSION TITLE
Time to Interactive is higher then the total loading time

### DISCUSSION THREAD

**Discussion by unisteven** [GoogleChrome/lighthouse#12261] [Q&A]
Hello!

For a research project I decided to use the lighthouse tool to examine the loading time of various frameworks/ methods. And when I was going through my results I noticed that the Time to Interactive is higher than the total load time. Which got my attention. 

So i decided to dig a little deeper, when surrounding the following code with a start and stop meassurement it was to my supprise to see that that value is way lower then the Time to Interactive.
``` 
            const start = new Date().getTime();
            const runnerResult = await lighthouse(site.url, options);
            const end = new Date().getTime();
```
![image](https://user-images.githubusercontent.com/6378043/111642546-79cf3e00-87fe-11eb-9a6a-b7497208e5f0.png)
How is it possible that it is able to calculate a result that took longer then the actual function itself.

If someone has any idea what is causing this/ where the time is based of I would be more than thankfull to know 😄

**Comment by connorjclark** [comment]
The metric values are a result of _simulated throttling_ (this is the default option, see our docs on how to change). We simulate a slow network (we build a graph of the page and its subresources and do some stuff in a system we call Lantern) instead of actually throttling the page load, so you don't have to wait as long. It also helps smooth out variance due to network speed differences.
